pkgs/applications/networking/cluster/cmctl/update.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env nix-shell
```

Comment:
And now we can replace this entire script with nix-update.

Reply:
> @superherointj I have added an `update.sh` which is based on the [fluxcd](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/cluster/fluxcd/update.sh) one. 

`fluxcd` update script is `customized` because the package required to an extra field (`manifestsSha256`). The current usual way to do an update script:

>  passthru.updateScript = nix-update-script { attrPath = pname; };

But I wonder if it covers the need of commit hash. If not, maybe this should be extended because it is a general use case, other packages need it too.

> Let me know if this is the approach we want to go in. 

I am more used to the manual / old school way (which is what you copied). I'd be fine about that. Still, as @SuperSandro2000 is suggesting something different. Maybe we learn something new? Let's see.

> I am quite keen on having the commit has in the version output to have consistency in the releases version.

Understood.

Reply:
> And now we can replace this entire script with nix-update.

Will the commit hash work as expected?

Reply:
Which ``manifestsSha256`` variable? I can't find it.

Reply:
> Which `manifestsSha256` variable? I can't find it.

No this. But commit hash.

Reply:
We don't need the commit hash. It is just informational and makes things harder.